core/input/input.cpp

```diff
@@ -1220,6 +1220,63 @@ void Input::set_event_dispatch_function(EventDispatchFunc p_function) {
 	event_dispatch_function = p_function;
 }
 
+bool Input::has_mapping(const StringName &p_guid) {
+	for (int i = 0; i < map_db.size(); i++) {
+		if (map_db[i].uid == p_guid) {
+			return true;
+		}
+	}
```

Comment:
```suggestion
	for (const JoyDeviceMapping map : map_db) {
		if (map.uid == p_guid) {
			return true;
		}
	}
```

Reply:
Thanks for the suggestion, but it seems like it should be `const JoyDeviceMapping&`.